app/Main.hs

```diff
@@ -145,3 +145,17 @@ main = do
     Just HapRollback -> rollback hapConfiguration
 
     Nothing -> hapHelpAction Nothing
+
+splitInColon :: String -> (String, String)
```

Comment:
@jsl @CristhianMotoche We could use something more general, like [`breakOn`](https://www.stackage.org/haddock/lts-5.5/extra-1.4.3/Data-List-Extra.html#v:breakOn) from extra:

```
breakOn :: Eq a => [a] -> [a] -> ([a], [a])
```

(We don't have to include the `extra` package, using `break` would be enough if that's all we need.)

I think using a list (and an empty list for no port) is fine here (we could turn it into a `Maybe` if we want to, but I don't think it makes a big difference).

We could use a `case` instead of a `let` and an `if`. Something like this:

```
case breakOn ':' server of
  (host, []) -> host
  (host, (_:port)) -> unwords [host, "-p", port]
```

The type signature I don't like is the one for `parseHostWithPort`. I think it would be better to make it

```
parseHostWithPort :: String -> String
```
 and use a second `fmap` (or `<$>`) above or anywhere else where a `Maybe` could make sense.